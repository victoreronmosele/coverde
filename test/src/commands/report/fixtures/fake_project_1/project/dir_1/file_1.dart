extension ExtendedNum on num {
  num plus(num x) => this + x;
  num minus(num x) => this - x;
}
