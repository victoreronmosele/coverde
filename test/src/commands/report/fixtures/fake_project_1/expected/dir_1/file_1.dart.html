<!DOCTYPE html><html lang="en"><head>
  <meta charset="UTF-8">
  <meta http-equiv="X-UA-Compatible" content="IE=edge">
  <meta name="viewport" content="width=device-width, initial-scale=1.0">
  <link rel="stylesheet" type="text/css" href="../report_style.css">
  <title class="headTitle">Coverage Report - lcov.info</title>
</head>

<body>
  <table width="100%" border="0" cellspacing="0" cellpadding="0">
    <tbody>
      <tr>
        <td class="title">Code Coverage Report</td>
      </tr>
      <tr>
        <td class="ruler" height="3px"></td>
      </tr>

      <tr>
        <td width="100%">
          <table cellpadding="1" border="0" width="100%">
            <tbody>
              <tr>
                <td width="10%" class="headerItem">Current view:</td>
                <td width="35%" class="headerValue currentFileName">
                  <a class="topLevelAnchor" href="../index.html">
                    top level
                  </a>
                  -
                  <a class="currentDirPath" href="index.html">test/src/commands/report/fixtures/fake_project_1/project/dir_1</a> - file_1.dart</td>
                <td width="5%"></td>
                <td width="15%"></td>
                <td width="10%" class="headerCovTableHead">Hit</td>
                <td width="10%" class="headerCovTableHead">Total</td>
                <td width="15%" class="headerCovTableHead">Coverage</td>
              </tr>
              <tr>
                <td class="headerItem">Test:</td>
                <td class="headerValue tracefileName">lcov.info</td>
                <td></td>
                <td class="headerItem">Lines:</td>
                <td class="headerCovTableEntry linesHit">1</td>
                <td class="headerCovTableEntry linesFound">2</td>
                <td class="covValue headerCovTableEntryLo">50.00 %</td>
              </tr>
              <tr>
                <td class="headerItem">Date:</td>
                <td class="headerValue lastTracefileModificationDate">2021-11-07 03:59:47.000</td>
                <td></td>
                <td></td>
                <td></td>
                <td></td>
                <td></td>
              </tr>
              <tr>
                <td height="3px"></td>
              </tr>
            </tbody>
          </table>
        </td>
      </tr>

      <tr>
        <td class="ruler" height="3px"></td>
      </tr>
    </tbody>
  </table>

  <table cellpadding="0" cellspacing="0" border="0">
    <tbody>
      <tr>
        <td><br></td>
      </tr>
      <tr>
        <td>
          <pre class="sourceHeading">          Line data    Source code</pre>
          <pre class="source"><a name="replace.with.line.number"><span class="lineNum">       1 </span><span class="sourceLine">            : extension ExtendedNum on num {</span>
</a><a name="replace.with.line.number"><span class="lineNum">       2 </span><span class="sourceLine lineCov">          2 :   num plus(num x) =&gt; this + x;</span>
</a><a name="replace.with.line.number"><span class="lineNum">       3 </span><span class="sourceLine lineNoCov">          0 :   num minus(num x) =&gt; this - x;</span>
</a><a name="replace.with.line.number"><span class="lineNum">       4 </span><span class="sourceLine">            : }</span>
</a></pre>
        </td>
      </tr>
    </tbody>
  </table>
  <br>

  <table width="100%" border="0" cellspacing="0" cellpadding="0">
    <tbody>
      <tr>
        <td class="ruler" height="3px"></td>
      </tr>
      <tr>
        <td class="versionInfo">
          Generated by:
          <a href="https://github.com/mrverdant13/coverde" target="_parent">
            coverde
          </a>
        </td>
      </tr>
    </tbody>
  </table>
  <br>



</body></html>